.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+<!--
+Use this template as a guide. Omit sections that don't apply.
+More details about the template are at https://github.com/edx/open-edx-proposals/pull/180
+-->
+
+## Description
+
+Describe what this pull request changes, and why. Include implications for people using this change.
+Design decisions should be documented in the repo (docstring / README / ADR) and linked here.
+
+Useful information:
+- Which edX user roles will this change impact? Common user roles are "Learner", "Course Author",
+"Developer" and "Admin".
+- How can it be tested?
```

Comment:
"How can it be tested" seems out of place in the "Description" section.

Reply:
As I noted earlier, I feel that "how can it be tested" is one of the most useful parts of the PR description. I fear that if it is relegated to "Other Information" in a middling bullet it will be omitted regularly. 

If it doesn't belong here, can we elevate it to the ## level?

Reply:
@pdpinch Can you clarify your position with a reference example of what information you expect from the "how can it be tested" section?  Also, what outcome are you expecting? Faster review cycle? Is the test instructions intended to be short-term just for the review cycle or longer-term for later reference?

Reply:
I wrote a "Testing instructions" section in the [PR description for this PR](https://github.com/edx/edx-platform/pull/25949#issue-546091511). 

Outcome: "Testing instructions" are primarily intended to speed up the review cycle, avoiding the reviewer wasting time trying to infer the steps to test from the description. Instructions can also be useful for later reference if a future developer is researching regression.

Reply:
I love testing instructions in PRs. It shows the author has thought carefully about the change and gives me a place to start.

Reply:
I do wonder if `How can it be tested` is the best wording. My suggestion might be, `Please provide step-by-step test instructions` (to be explicit that we want this, and what we are looking for)

Reply:
I've moved testing instructions to a top-level section.